_ml-commons-plugin/cluster-settings.md

```diff
@@ -15,7 +15,7 @@ This page provides an overview of `opensearch.yml` settings that can be configur
 ### Setting
 
 ```
-plugins.ml_commons.only_run_on_ml_node: false
```

Comment:
Not for this line. 
Seems these system setting missed
1. `plugins.ml_commons.max_ml_task_per_node: 10`:

Description: How many ML task can run on per node. Set as `0` means no ML task allowed to run.

Default value: `10`, 
Value range: [0, 10000]

2. `plugins.ml_commons.max_model_on_node: 10`:

Description: how many models can loaded to one ML node. If set as 0, then can't load any model.

Default value: `10`, 
Value range: [0, 10000]

3. (new setting in 2.5) `plugins.ml_commons.ml_task_timeout_in_seconds: 600`

Description: How long the ML task will live. By default it will time out after 10minutes. Then task will be set as FAILED.

Default value: `600`, 
Value range: [1, 86400]

4. (new setting in 2.5) `plugins.ml_commons.native_memory_threshold: 90`

Description: We add a new circuit breaker in 2.5 which will check all system memory usage (not just JVM heap usage) before running ML task. If it exceeds the threshold, will throw exception and stop running the ML task. Set as `0` means no ML task allowed to run, set as `100` means close the system memory circuit breaker.

Default value: `90`, 
Value range: [0, 100]

Reply:
I'll address these in separate PRs.

1. For the new ML settings in 2.5.
2. To add the ones we missed in 2.4.